src/vue/backend/src/components/agentCard.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <checkbox-label
+        :option="option"
+        customClass="agent-card"
+        @checked="$emit('agent-selected', [$event.data, agent.id])"
+    >
+        <template v-slot:content>
+            <idx-block :className="{
+                    'agent-card__image-wrap': true,
+                    'agent-card__image--placeholder': !primaryImage
+                }">
+                <img v-if="primaryImage" :src="agent.image">
+                <div v-else>
+                    {{ agentInitials }}
+                </div>
+            </idx-block>
+            <idx-block className="agent-card__content">
+                <idx-block className="agent-card__name">{{ agent.name }}</idx-block>
+                <idx-block className="agent-card__title">{{ agent.title }}</idx-block>
+                <idx-block className="agent-card__email">{{ agent.email }}</idx-block>
+                <idx-block className="agent-card__id">#{{ agent.id }}</idx-block>
+            </idx-block>
+            <idx-block v-if="agent.imported" className="agent-card__import-flag">
+                <idx-block className="agent-card__imported">
+                    Imported <img :src="check">
+                </idx-block>
+                <idx-block @click.native.stop="$emit('removeAgent', agent.id)" className="agent-card__delete">
+                    <img :src="deleteIcon">
+                </idx-block>
+            </idx-block>
+        </template>
+    </checkbox-label>
+</template>
+<script>
+import deleteIcon from '@/assets/trash-light.svg'
+import check from '@/assets/check-light-white.svg'
+export default {
+    name: 'agent-card',
+    data () {
+        return {
+            option: {
+                value: 'selected',
+                label: 'selected'
+            },
+            deleteIcon,
+            check
+        }
+    },
+    props: {
+        agent: {
+            type: Object,
+            default: () => {}
+        }
+    },
+    computed: {
+        primaryImage () {
+            return this.agent.image !== (null || '')
```

Comment:
Could do a falsey check here `!this.agent.image` since it'd probably want to return false for undefined as well